src/components/curriculum/elements/AddBar.vue

```diff
@@ -37,12 +37,10 @@
 		margin: 28px 0;
 		max-width: 200px;
 		.incrementor {
-			display: block;
-			margin-top: var(--gutter);
+			display: flex;
```

Comment:
[DÚVIDA] É uma boa prática manter o HTML e o CSS juntos, qual os prós e contras?

Reply:
Na minha opinião manter HTML e CSS juntos fere o design pattern SoC (Separation of Concerns) caso isso seja adotado no projeto, mas não há certo nem errado, é questão de padrões de projeto. Eu prefiro manter em arquivos separados e aqui vai prós e contras do que eu penso sobre:

- Prós:
  - Separação de responsabilidades;
  - Organização de componentes.
 
- Contras:
  - Dificulta nos primeiros momentos a pesquisa do arquivo CSS no editor.

Reply:
@rntvicente 

Quanto a manter HTML e CSS juntos é uma questão de design do framework: os famosos Vue Single File Components. Neste link eles dão uma boa overview dos motivos de ser assim https://vuejs.org/v2/guide/single-file-components.html.

Particularmente eu gosto bastante disso NO VUE. Mas se eu fosse fazer uma página sem framework e sem componentização, eu separaria todos os concerns como o @thdq levantou ali, com ctz. Mesmo em Vue é importante separar concerns: o estilo que está no component diz respeito unicamente ao component, não deve haver estilos globais ali, entao vc tem duas camas: Glogal Style e Scoped Style.